dev.yml

```diff
@@ -3,7 +3,7 @@ name: tapioca
 type: ruby
 
 up:
-  - ruby: '3.1'
+  - ruby: '3.1.2'
```

Comment:
> With the patch, Dev downloads a precompiled binary, but without it seems to choose to compile from source, which seems unnecessary.

Should we add this as a comment explaining why we use this version?

Reply:
[Searching for `x.y` Ruby versions in `dev.yml` across our org](https://github.com/search?q=org%3AShopify+path%3A%2Fdev.yml%24%2F+%2Fruby%3A+%5B%22%27%5D%5Cd%5C.%5Cd%5B%22%27%5D%2F&type=code) only turns up 5 matches, so I think it's safe to say this is uncommon.